lib/inspec/runner.rb

```diff
@@ -270,5 +273,19 @@ def register_rule(rule)
 
       examples.each { |e| @test_collector.add_test(e, rule) }
     end
+
+    def validate_attributes_file_readability!(target)
+      return if File.readable?(target)
```

Comment:
While this is technically correct, it's an awkward implementation because this line is 100% connected to the exception at L286. If there were 3+ different conditions we wanted to check for in order to provide a unique error message for each, then this might be fine.

Please remove this line and wrap the exception at L286 with an `unless` block like the one above it so it's 100% clear the condition of that exception.

Reply:
Can do! I'll have it return `true` to avoid the rubocop `Style/GuardClause` offense.